honeybadger/config.py

```diff
@@ -15,7 +15,8 @@ class Configuration(object):
         ('params_filters', list),
         ('force_report_data', bool),
         ('force_sync', bool),
-        ('excluded_exceptions', list)
+        ('excluded_exceptions', list),
+        ('is_using_local_variables', bool)
```

Comment:
How about a `report_local_variables` or `include_local_variables`?

Reply:
@subzero10, made the changes 👍